graveyard.json

```diff
@@ -2342,5 +2342,13 @@
     "link": "https://9to5google.com/2023/09/28/google-jamboard/",
     "description": "Google Jamboard was a web and native whiteboard app that offered a rich collaborative experience.",
     "type": "app"
+  },
+  {
+    "name": "Photo Sphere",
+    "dateOpen": "2012-11-13",
+    "dateClose": "2023-10-04",
+    "link": "https://www.androidauthority.com/google-photo-sphere-mode-pixel-8-3373610/",
+    "description": "Photo Sphere mode was introduced in 2012 for Android 4.2 with the Nexus 4 launch, enabling 360° photo capture by stitching multiple panorama shots; it was later dropped due to user experience issues.",
```

Comment:
> it was later dropped due to user experience issues

Is that last bit actually the reason? I don't know that there's a source making that claim authoritatively. It would be more factual to just say it was dropped during the Pixel 8 launch.

Reply:
When that's an assumption but as per my experience that is a good assumption

Reply:
Although I have updated the issue you have highlighted.